### PR TITLE
Lpiet-rdam

### PR DESCRIPTION
**Pull request by lpiet**
Adding a basic file for rotterdam, nl with food, attractions and bars.

### CODE REVIEW

File: lpiet-rdam.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Rdam, NL
+
+## Eat
+- Mc
+- Bk
+- Smullers 
+
+## Drink
+- Breakaway :beers:
```
Comment (cshawnbrown) [inline lpiet-rdam.md:9]: No comments really

File: lpiet-rdam.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Rdam, NL
```
Comment (cshawnbrown) [inline lpiet-rdam.md:1]: Just testing comments and reviews

**Review by cshawnbrown — CHANGES_REQUESTED** [review]
@lpiet  Please review

File: lpiet-rdam.md (hunk 2 above)
Comment (lpiet) [reply lpiet-rdam.md:1]: seems to be working

**Comment by lpiet** [comment]
no changes requested so nothing to do, could be set to allowed?

**Pull request merged**